javascript-light-bulb/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title> Javascript Lightbulb </title>
+  <link href="styles.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
```

Comment:
I like that you're trying to use more semantic elements! I think a better place for `main` would be on line 10. All of these `div` elements would technically be apart of the main content.